src/main/java/pl/winiarczyk/lesson4/homework/AgeVerify.java

```diff
@@ -1,16 +0,0 @@
-package pl.winiarczyk.lesson4.homework;
-
-public class AgeVerify {
-
-  public static boolean isMature(int age) {
-    return age >= 18;
-  }
-
-  public static void main(String[] args) {
```

Comment:
A czemu to kasujesz? 

![image](https://user-images.githubusercontent.com/28653713/132904001-2efe11a6-b943-40be-85b1-bfea1f2132e4.png)


W tym commicie powinno być tutaj 1 ;) dodajesz jeden plik z zadaniem.